Build fails due to error  Could not find com.google.firebase:firebase-crashlytics-ktx:.

### Overview

Hi, I am trying to build the main version from cloned from github on 13 Sep.

But the build is failing with the error:

 Could not find com.google.firebase:firebase-crashlytics-ktx:.

Full error message once I select Build -> Make Project is:
```
Configuration cache state could not be cached: field `provider` of `org.gradle.configurationcache.serialization.codecs.ProviderBackedFileCollectionSpec` bean found in field `element` of `java.util.Collections$SingletonList` bean found in field `elements` of `org.gradle.configurationcache.serialization.codecs.ResolutionBackedFileCollectionSpec` bean found in field `__classpathSnapshot__` of `org.jetbrains.kotlin.gradle.tasks.KotlinCompile$ClasspathSnapshotProperties` bean found in field `__classpathSnapshotProperties__` of task `:logging:crashlytics:compileDebugKotlin` of type `org.jetbrains.kotlin.gradle.tasks.KotlinCompile`: error writing value of type 'org.gradle.api.internal.provider.DefaultProvider'
> Could not resolve all files for configuration ':logging:crashlytics:debugCompileClasspath'.
   > Could not find com.google.firebase:firebase-crashlytics-ktx:.
     Required by:
         project :logging:crashlytics

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

```

Are  there any  project specific steps required for crashlytics for Voice?
Or should I search for generic solution for this problem?

thanks
Linas

### How to reproduce

1. git clone the project
2. open the project in Android Studio
3. Build
4. Make Project

### Version

main

### Android Version

dev

### Device Model

dev

Additionally, I have also tried 
setting the following in `voice.yml`:
```
VOICE_USE_PROPRIETARY_LIBRARIES: "false"
```

And I am still getting the same result (this time building from command line):
```
bro1@bro1-IdeaPad-Flex-5-15ALC05:~/projects/Voice$ ./gradlew build
Calculating task graph as no configuration cache is available for tasks: build
Type-safe project accessors is an incubating feature.

FAILURE: Build failed with an exception.

* What went wrong:
Configuration cache state could not be cached: field `provider` of `org.gradle.configurationcache.serialization.codecs.ProviderBackedFileCollectionSpec` bean found in field `element` of `java.util.Collections$SingletonList` bean found in field `elements` of `org.gradle.configurationcache.serialization.codecs.ResolutionBackedFileCollectionSpec` bean found in field `__classpathSnapshot__` of `org.jetbrains.kotlin.gradle.tasks.KotlinCompile$ClasspathSnapshotProperties` bean found in field `__classpathSnapshotProperties__` of task `:logging:crashlytics:compileDebugKotlin` of type `org.jetbrains.kotlin.gradle.tasks.KotlinCompile`: error writing value of type 'org.gradle.api.internal.provider.DefaultProvider'
> Could not resolve all files for configuration ':logging:crashlytics:debugCompileClasspath'.
   > Could not find com.google.firebase:firebase-crashlytics-ktx:.
     Required by:
         project :logging:crashlytics

```

--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -15,7 +15,7 @@ plugins {
   alias(libs.plugins.playPublish)
 }
 
-val useProprietaryLibraries = providers.environmentVariable("VOICE_USE_PROPRIETARY_LIBRARIES").orNull == "true"
+val useProprietaryLibraries = System.getenv("VOICE_USE_PROPRIETARY_LIBRARIES") == "true"
 if (useProprietaryLibraries) {
   pluginManager.apply(libs.plugins.crashlytics.get().pluginId)
   if (file("google-services.json").exists()) {
@@ -180,7 +180,7 @@ dependencies {
     implementation(libs.firebase.crashlytics)
     implementation(libs.firebase.analytics)
     implementation(libs.firebase.remoteconfig)
-    implementation(projects.logging.crashlytics)
+    implementation(project(":logging:crashlytics"))
     implementation(project(":review:play"))
   } else {
     implementation(projects.review.noop)

--- a/plugins/src/main/kotlin/baseSetup.kt
+++ b/plugins/src/main/kotlin/baseSetup.kt
@@ -59,7 +59,7 @@ fun Project.baseSetup() {
       add("implementation", project(":logging:core"))
     }
     add("implementation", platform(libs.findLibrary("compose-bom").get()))
-    if (providers.environmentVariable("VOICE_USE_PROPRIETARY_LIBRARIES").orNull == "true") {
+    if (System.getenv("VOICE_USE_PROPRIETARY_LIBRARIES") == "true") {
       add("implementation", platform(libs.findLibrary("firebase-bom").get()))
     }
     add("androidTestImplementation", platform(libs.findLibrary("compose-bom").get()))

--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -61,10 +61,10 @@ include(":migration")
 include(":scripts")
 include(":logging:core")
 include(":logging:debug")
-include(":logging:crashlytics")
 include(":documentfile")
 include(":onboarding")
-if (!providers.gradleProperty("foss").isPresent) {
+if (System.getenv("VOICE_USE_PROPRIETARY_LIBRARIES") == "true") {
+  include(":logging:crashlytics")
   include(":review:play")
 }
 include(":review:noop")

